packit/utils/koji_helper.py

```diff
@@ -268,6 +262,51 @@ def remove_sidetag(self, sidetag: str) -> None:
         except Exception as e:
             logger.debug(f"Failed to remove sidetag {sidetag} in Koji: {e}")
 
+    def get_build_target(self, dist_git_branch: str) -> Optional[dict]:
+        """
+        Gets a build target from a dist-git branch name.
+
+        Args:
+            dist_git_branch: dist-git branch name.
+
+        Returns:
+            Build target or None if not found.
+        """
+        target_name = self.get_build_target_name(dist_git_branch)
+        try:
+            target = self.session.getBuildTarget(target_name)
+        except Exception as e:
```

Comment:
Is this a generic exception just for the ease of testing?

Reply:
It's `Exception` because noone ever knows what Koji API will throw on them :sweat_smile:
In this particular case, judging by the [docs](https://koji.fedoraproject.org/koji/api), it doesn't even throw an exception and only returns `None`, but I don't trust Koji API and I think it's better to be safe.